### DISCUSSION TITLE
Pagination instead of infinite scroll

### DISCUSSION THREAD

**Discussion by erichhaemmerle** [Pagefind/pagefind#131] [Q&A]
Is there a straightforward way to allow the results to be paginated when using the Pagefind UI or can it only be infinite scroll?

**Comment by bglw** [comment]
Hello 👋 

Not from the Pagefind UI, no. I want to avoid having a long list of options to digest when using the UI, so I'm looking to instead offer multiple UI libraries for different use-cases — in which case one of those available libraries could be a paginated widget.

If you have any experience with Svelte, stealing the Pagefind UI code and changing it to a pagination model would be your best bet bet right now.
